src/java_bytecode/java_bytecode_parser.cpp

```diff
@@ -946,6 +947,12 @@ void java_bytecode_parsert::rmethod_attribute(methodt &method)
     {
       u2 start_pc=read_u2();
       u2 end_pc=read_u2();
+
+      INVARIANT(
```

Comment:
would you mind adding that this invariant is from the class file format spec ("4.7.3. The Code Attribute" for Java8)